### ISSUE TITLE
XLNet: Generalized Autoregressive Pretraining for Language Understanding

### ISSUE THREAD

**Issue by jinmang2** [jiphyeonjeon/season3#26]
## 집현전 중급반 스터디
- 2022년 5월 22일 일요일 9시
- 김찬우님 박창현님 서범진님 홍범님 발표
- 논문 링크: https://arxiv.org/abs/1906.08237

> ### Abstract
> With the capability of modeling bidirectional contexts, denoising autoencoding based pretraining like BERT achieves better performance than pretraining approaches based on autoregressive language modeling. However, relying on corrupting the input with masks, BERT neglects dependency between the masked positions and suffers from a pretrain-finetune discrepancy. In light of these pros and cons, we propose XLNet, a generalized autoregressive pretraining method that (1) enables learning bidirectional contexts by maximizing the expected likelihood over all permutations of the factorization order and (2) overcomes the limitations of BERT thanks to its autoregressive formulation. Furthermore, XLNet integrates ideas from Transformer-XL, the state-of-the-art autoregressive model, into pretraining. Empirically, under comparable experiment settings, XLNet outperforms BERT on 20 tasks, often by a large margin, including question answering, natural language inference, sentiment analysis, and document ranking.

**Comment by kco4776** [comment]
10조 발표자료 입니다!
https://drive.google.com/file/d/1bTV3_wYX894WL02Gf9OtieXz0FBcP5Jw/view?usp=sharing

**Comment by jinmang2** [comment]
- 발표 링크: https://www.youtube.com/watch?v=Oy3Jv9P7i5w&list=PL2tRglRS_GqjCcBVM26gYNdoOnX12lLIh&index=8
